_posts/2020-04-17-dask-summit.md

```diff
@@ -0,0 +1,353 @@
+
+
+5: Performance and Tooling
+--------------------------
+
+#### Berkeley Institute for Data Science
+
+-   Title: Numpy APIs
+-   Presenters: Sebastian Berg
+
+#### Ursa Labs
+
+-   Title: Arrow
+-   Presenters: Joris Van den Bossche
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vQY3ubjCFkMcU_b8p2xmuXN8VVR1BxxSWZDe5Vy-ftnH2CstZILvTo2pRBv5R_VDk85rNjVoWew2AJl/embed?start=false&loop=false&delayms=3000" frameborder="0" width="600" height="366" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
+
+#### NVIDIA
+
+-   Title: RAPIDS
+-   Presenters: Keith Kraus
+-   Also attending:
+    -   Mike Beaumont
+    -   Richard Zamora
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vQiNrupzQlSqsu95AAHqIhU1V_iVUav_0WlIp4dXdSE6Izze1BL8mkFbIzg7p8CndEi9bjWaC2OVlyu/embed?start=false&loop=false&delayms=3000" frameborder="0" width="600" height="366" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
+
+#### NVIDIA
+
+-   Title: UCX
+-   Presenters: Ben Zaitlen
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vRU-vsXsnXgeLKdmtWZkZVV_-mOojsNesCbQKJgmWkwSjxj5ZdwkmS6X4tOt3HpFrIOfNROSlV_8l84/embed?start=false&loop=false&delayms=3000" frameborder="0" width="600" height="366" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
+
+6: Xarray
+---------
+
+#### USGS and NCAR
+
+-   Title: Dask in Pangeo
+-   Presenters: Rich Signell and Anderson Banihirwe
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vStqGiQy6pJDYhRgF-BZylQussINK5BGlhnidOVCUECo_ebYqRH9cSY4e-2z7BfFFvTfvkqq_M1jXBX/embed?start=false&loop=false&delayms=3000" frameborder="0" width="600" height="366" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
+
+#### LBNL
+
+-   Title: Accelerating Experimental Science with Dask
+-   Presenters: Matt Henderson
+-   [Slides](https://drive.google.com/file/d/1DVVzYmhkDhO2xs0tmxpPCkxx5c4o63bO/view) - Fill too large to preview
+
+#### LANL
+
+-   Title: Seismic Analysis
+-   Presenters: Jonathan MacCarthy
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vSWAgKLxt1tBZxXjQfIRQNFPvAMFYZ-z0hkMy7euPnOHwO9pomH_gM8cKUTKXA68w/embed?start=false&loop=false&delayms=3000" frameborder="0" width="600" height="404" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
+
+
+Unstructured Time
+-----------------
+
+Having rapid fire talks in the morning, followed by unstructured time in the
+afternoon was a productive combination that we would recommend to other broad
+groups in the future.  Engagement and productivity was really high throughout
```

Comment:
> followed by unstructured time in the
afternoon was a productive combination that we would recommend to other broad
groups in the future.

My first thought: what happened in the unstructured time? Maybe this reorganization would be slightly better:

> Having rapid fire talks in the morning, followed by unstructured time in the afternoon was a productive combination. Below you'll see pictures from geo-scientists and quants talking about the same challenges.  You'll also see library maintainers from Pandas/Arrow/RAPDIS/Dask all working together on joint solutions.
>
> [images]
>
> This unstructured time is a productive combination that we would recommend to other technically diverse groups in the future. Engagement and productivity was really high throughout the workshop.

Reply:
Done.  Thanks @stsievert !